src/jdk.jdeps/share/man/jnativescan.1

```diff
@@ -0,0 +1,215 @@
+.hy
+.SH NAME
+.PP
+jnativescan - static analysis tool that scans one or more jar files for
+uses of native functionality, such as restricted methods or
+\f[V]native\f[R] method declarations.
+.SH SYNOPSIS
+.PP
+\f[V]jnativescan\f[R] [\f[I]options\f[R]]
+.TP
+\f[I]options\f[R]
+See \f[B]Options for the jnativescan Command\f[R]
+.SH DESCRIPTION
+.PP
+The \f[V]jnative\f[R] tool is a static analysis tool provided by the JDK
+that scans a JAR file for uses of native functionality, such as
+restricted methods or \f[V]native\f[R] method declarations.
+.PP
+\f[V]jnativescan\f[R] accepts a runtime class path and module path
+configuration, as well as a set of root modules, and a target release.
+It scans the jars on the class and module paths, and reports uses of
+native functionality either in a tree like structure, which also
+identifies that calling classes and methods, or as a list of module
+names when the \f[V]--print-native-access\f[R] flag is specified.
+.SH OPTIONS FOR THE JNATIVESCAN COMMAND
+.PP
+The following options are available:
+.TP
+\f[V]--class-path\f[R] \f[I]path\f[R]
+Used to specify a path list of class path jar files to be scanned.
+All jar files specified through this path will be scanned.
+If a jar file contains a \f[V]Class-Path\f[R] attribute in its manifest,
+jar files listed there will be scanned as well.
+Jar files listed in the \f[V]Class-Path\f[R] manifest attribute that can
+not be found are ignored.
+.TP
+\f[V]--module-path\f[R] \f[I]path\f[R]
+Used to specify a path list of jar files or directories containing
+module path jar files.
+The list jar files that will be scanned depends on the
+\f[V]--add-modules\f[R] option.
+.RS
+.PP
+For both the \f[V]--class-path\f[R] and \f[V]--module-path\f[R] options,
+\f[I]path\f[R] should be a search path that consists of one or more jar
+files, separated by the system-specific path separator.
+For example:
+.IP \[bu] 2
+\f[B]Linux and macOS:\f[R]
+.RS 2
+.RS
+.PP
+\f[V]--class-path /some/foo.jar:/another/different/bar.jar\f[R]
+.RE
+.RE
+.PP
+\f[B]Note:\f[R]
+.PP
+On Windows, use a semicolon (\f[V];\f[R]) as the separator instead of a
+colon (\f[V]:\f[R]).
+.IP \[bu] 2
+\f[B]Windows:\f[R]
+.RS 2
+.RS
+.PP
+\f[V]--class-path C:\[rs]some\[rs]foo.jar;C:\[rs]another\[rs]different\[rs]bar.jar\f[R]
+.RE
+.RE
+.RE
+.TP
+\f[V]--add-modules\f[R] \f[I]module[,module...]\f[R]
+Used to specifiy a comma-separated list of module names that indicate
+the root modules to scan.
+All the modules in the root set will be scanned, as well as any modules
+that they depend on.
+This includes dependencies on service implementations specified through
+the \f[V]uses\f[R] directive in a module\[aq]s \f[V]module-info\f[R]
+file.
+All modules found on the module path that provide an implementation of
+such a service will be scanned as well.
+.TP
+\f[V]--release\f[R] \f[I]version\f[R]
+Used to specify the Java SE release that specifies the set of restricted
```

Comment:
In principle, the release could also affect which methods are native or not?

Reply:
Yes

Reply:
You are right... declarations inside the JDK are... inside the JDK